postgres/assets/monitors/percent_usage_connections.json

```diff
@@ -19,13 +19,7 @@
 		"include_tags": true,
 		"thresholds": {
 			"critical": 0.9,
-			"warning": 0.8,
-			"critical_recovery": 0.9,
-			"warning_recovery": 0.8
-		},
-		"threshold_windows": {
-			"trigger_window": "last_15m",
-			"recovery_window": "last_15m"
```

Comment:
`threshold_windows` only works for anomaly monitor (this one is metric)

critical/warning_recovery should have a different number.